lib/src/command.dart

```diff
@@ -207,7 +221,7 @@ and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
   /// appropriate exit code could be found.
   int _chooseExitCode(exception) {
     if (exception is SolveFailure) {
-      var packageNotFound = exception.packageNotFound;
+      dynamic packageNotFound = exception.packageNotFound;
```

Comment:
I don't think this should be `dynamic`.
Perhaps:
```suggestion
      PackageNotFoundException? packageNotFound = exception.packageNotFound;
```